challenges/Day_9/README.md

```diff
@@ -0,0 +1,18 @@
+TO GET STARTED:
+1. Open Terminal and cd into the back directory
+2. run: npm install
+3. run: node app.js
+3. open a new terminal tab (ctrl+shift+T)
+4. cd into the front directory
+5. run: npm install
+6. run: npm start 
+
+
+
+There are two files, one for the frontend, and one for the backend
+The frontend has a forum that will be sent to the backend at url: 127.0.0.1:3007
+The backend will send that name right back and can be viewed in the console.
+Your job is:
+1. Add another input field for age
+2. Create an if statement in the backend to only send a response back if age is greater than 15
```

Comment:
I like your idea about using an IF statement in the backend to only send if age > 15. 

this isn't typically done though, maybe we can send another property like: isAdult, in addition to sending the age
so... `Send two properties on the response: (1) the age of the person that was sent in the request, and (2) whether the person is an adult (older than 18)`